frontend/src/components/Cards/Card/Index.jsx

```diff
@@ -0,0 +1,50 @@
+import "../../../services/reset.css";
```

Comment:
Import the reset.css once (in App.jsx, or better yet in main.jsx)

Reply:
done